e2e/content-services/tag-component.e2e.ts

```diff
@@ -88,16 +81,13 @@ describe('Tag component', () => {
     });
 
     afterAll(async (done) => {
-        try {
-            await uploadActions.deleteFilesOrFolder(this.alfrescoJsApi, pdfUploadedFile.entry.id);
-        } catch (error) {
-        }
+        await uploadActions.deleteFileOrFolder(pdfUploadedFile.entry.id);
+        browser.refresh();
```

Comment:
Why you changed this into a browser refresh? i think what it was doing was different